docs/src/release-notes-python.md

```diff
@@ -4,6 +4,47 @@ title: "Release notes"
 toc_max_heading_level: 2
 ---
 
+## Version 1.51
+
+### Highlights
+
+* New option [`option: BrowserContext.storageState.indexedDB`] for [`method: BrowserContext.storageState`] allows to save and restore IndexedDB contents. Useful when your application uses [IndexedDB API](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) to store authentication tokens, like Firebase Authentication.
+
+  Here is an example following the [authentication guide](./auth.md#reusing-signed-in-state):
+
+  ```python
+  # Save storage state into the file. Make sure to include IndexedDB.
+  storage = await context.storage_state(path="state.json", indexedDB=True)
```

Comment:
lets wait with landing this until we have the Python roll but I think it should be `indexed_db`

Reply:
It is indeed, according to [playwright.dev](https://playwright.dev/python/docs/next/api/class-browsercontext#browser-context-storage-state-option-indexed-db).